configs/datasets/single_episode.yaml

```diff
@@ -0,0 +1,2 @@
+DATASET:
+  NUM_EPISODE_SAMPLE: 1
```

Comment:
Ideally, updating this should be done via specifying `DATASET.NUM_EPISODE_SAMPLE 1`, which should be specified in documentation and then you don't even need this config. Even better, add a `test_run` argument which just loads this configuration automatically.

Reply:
Agreed. Creating a separate config file (with only 2 lines) just for this functionality seems clunky. I think without any changes or any additional config file, this config should be able to be passed by adding `opts=[DATASET.NUM_EPISODE_SAMPLE, 1]` to the command line call (according to [this documentation](https://github.com/facebookresearch/habitat-api/blob/f1c6922df77a3224107c7f27bb0b40e34e9f253e/habitat/config/default.py#L175-L177)). However, I don't think it's working right now. Alternatively, I can add a `test-run` argument, which will either require the two-liner yaml file or require additional modification in training script. Will discuss with Oleksandr.

Reply:
The way to do it in yacs is that you just add `DATASET.NUM_EPISODE_SAMPLE 1` to the end of your command.

Reply:
Discussed with Amanpreet through chat. We are missing something like [this](https://github.com/facebookresearch/pythia/blob/fd2c9a52cbb983488fe62f1c4ec8719070bd07d0/pythia/utils/flags.py#L224-L229). Should work once we add this.

Reply:
Exactly `get_config` function has this option, but we need to pass arguments there. Makes sense to add such option to `train_ppo.py`.